### ISSUE TITLE
Format Options screen with grey selection 

### ISSUE THREAD

**Issue by ccdicamillo** [WycliffeAssociates/quick_usfm_converter#57]
It would be great if possible to show the option selected in grey (instead of white) as it may look to the end-user (as it did to me!) that the option selected was in grey.

**Comment by ChrisJarka** [comment]
I agree - I was confused also. Each time I came to these decisions, I referred to the 3 choices for font size to remind me what color the selected button was.

**Comment by ccdicamillo** [comment]
Hi Bryan, (Chris) -

Thanks so much for working on this! Yes, I think it is better (was the text
changed to grey?) Chris, your thoughts?

Best regards,
Charla

**Comment by ccdicamillo** [comment]
Oh, I didn't see your response Chris...:)
